mergebot-config.json

```diff
@@ -0,0 +1,10 @@
+{
+   "update_jira_on_merge": {
+     "enabled": true,
+     "interactive": false,
+     "fix_version_map": {
+       "master": "K8s Base Addons 2.1.0",
+       "testing": "K8s Base Addons 2.0.0",
```

Comment:
this needs constantly be updated to reflect the next to-be-released version. @joejulian can you handle that (or just add it to your release automation?) 🙇